test/integration_test.tsx

```diff
@@ -487,6 +487,29 @@ describe('integration tests', () => {
       assert.equal(output, '<div><Component><p>foo</p></Component></div>');
     });
 
+    it('renders children of components rendered without JSX', () => {
```

Comment:
I'm not sure if this is necessarily specific to not using JSX.

Reply:
You're might be right. I just ended up going from the specific case that had happened to me (took me a while to figure it out 😅)

Thanks for taking a look into this so quickly though!